library_generation/DEVELOPMENT.md

```diff
@@ -45,35 +44,43 @@ python -m unittest discover -s test/ -p "*unit_tests.py"
 # Running the scripts in your local environment
 
 Although the scripts are designed to be run in a Docker container, you can also
-run them directly. This section explains how to run the entrypoint script
+run them directly.
+This section explains how to run the entrypoint script
 (`library_generation/cli/entry_point.py`).
 
-## Installing prerequisites
+## Assumptions made by the scripts
+### The Hermetic Build's well-known folder
+Located in `${HOME}/.library_generation`, this folder is assumed by the scripts
+to contain the generator JAR. 
+Please note that this is a recent feature and only this jar is expected to be
+there. 
+Soon enough, more binaries such as the gRPC and protoc plugins will also be
```

Comment:
I would prefer not mention this as this is not implemented yet. We may treat gRPC and protoc differently as well.

Reply:
Got it, I removed this sentence.